solidity/artifacts/DKG.sol/DKG.json

```diff
@@ -0,0 +1,367 @@
+{
```

Comment:
Is this addition intentional? Given that the project is intended to build this from the `.sol` file I don't see a reason to check it in. Also having this checked in opeens up the possibility that the `.sol` file can diverge from the `json`.